### PR TITLE
Chore/add openapi jackson and jpa config

### PR DESCRIPTION
**Pull request by YukinaMochizuki**
Added springdoc-openapi-ui to automatically generate OpenAPI specs and Swagger UI pages and fix coding style issues.

### CODE REVIEW

File: src/main/java/io/charkchalk/backend/entity/range/DateRange.java [hunk 1]
```diff
@@ -22,14 +23,16 @@ public class DateRange {
     @Column(name = "id", nullable = false)
     private Long id;
 
+    @Column(name = "name")
```
Comment (moontai0724) [inline src/main/java/io/charkchalk/backend/entity/range/DateRange.java]: `UNIQUE` or not?

Comment (YukinaMochizuki) [reply src/main/java/io/charkchalk/backend/entity/range/DateRange.java]: I think that's for sure

File: src/main/java/io/charkchalk/backend/entity/range/TimeRange.java [hunk 2]
```diff
@@ -23,7 +26,7 @@ public class TimeRange {
     private Long id;
```
Comment (moontai0724) [inline src/main/java/io/charkchalk/backend/entity/range/TimeRange.java:27]: Will need a `name` or not?

File: src/main/java/io/charkchalk/backend/configuration/OpenAPIConfig.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+@Configuration
+public class OpenAPIConfig {
+    @Bean
+    public OpenAPI openAPI() {
+        return new OpenAPI()
+                .info(new Info().title("Charkchalk Backend API")
+                        .description("Charkchalk Backend API implemented with Spring Boot RESTful service " +
+                                "and documented using springdoc-openapi and OpenAPI 3.")
+                        .version("v0.0.1")
+                        .license(new License().name("Apache 2.0")
```
Comment (moontai0724) [inline src/main/java/io/charkchalk/backend/configuration/OpenAPIConfig.java]: Leave a note here, currently is unlicensed.

**Review by moontai0724 — APPROVED** [review]

File: src/main/java/io/charkchalk/backend/entity/range/TimeRange.java (hunk 2 above)
Comment (YukinaMochizuki) [reply src/main/java/io/charkchalk/backend/entity/range/TimeRange.java:27]: In extreme cases, the course time is likely to conflict with the schedule; based on this consideration; there are currently no plans to name the `TimeRange`.

File: src/main/java/io/charkchalk/backend/configuration/OpenAPIConfig.java (hunk 3 above)
Comment (YukinaMochizuki) [reply src/main/java/io/charkchalk/backend/configuration/OpenAPIConfig.java]: I will delete it

**Review by moontai0724 — APPROVED** [review]

**Pull request merged**